bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/hbasesnapshots/EndToEndIT.java

```diff
@@ -126,6 +158,17 @@ private static String getTestProperty(String name) {
 
   @After
   public void teardown() throws IOException {
+    final List<GcsPath> paths = gcsUtil.expand(GcsPath.fromUri(syncTableOutputDir + "/*"));
+
+    if (!paths.isEmpty()) {
+      final List<String> pathStrs = new ArrayList<>();
+
+      for (GcsPath path : paths) {
+        pathStrs.add(path.toString());
+      }
+      this.gcsUtil.remove(pathStrs);
+    }
+
```

Comment:
This doesnt have to happen in this PR, so maybe just add a TODO. But we've had issues in the past of failing to cleanup resources during teardown. For example the test timing out and teardown never running. A effective way to avoid such issues is to embed a timestamp in your resource names and have setup check for orphans from previous runs and clean them up:
https://github.com/googleapis/java-bigtable/blob/35588d89b9b243eb691a29d3aff16b9f5a08fbb8/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/AbstractTestEnv.java#L108-L119

Reply:
done.